### PR TITLE
Add description for EditorPlugin.update_overlays

### PR DESCRIPTION
**Pull request by volzhs**

### CODE REVIEW

File: doc/classes/EditorPlugin.xml [hunk 1]
```diff
@@ -471,6 +471,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Updates the overlays of the [2D/3D] viewport.
```
Comment (mhilbrunner) [inline doc/classes/EditorPlugin.xml]: I'm not sure if the square brackets will work here, I'd need to build the docs to check. Usually they are part of the markup. If you're sure this works, ignore me, if not, maybe change to "(2D/3D)" or escape the square brackets with a backslash?

Comment (volzhs) [reply doc/classes/EditorPlugin.xml]: I would change it to "(2D/3D)". thanks. :)

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Looks good, thanks :)

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**